smtk-import/CMakeLists.txt

```diff
@@ -92,8 +92,7 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
 if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
         set(SMTK_LINK_LIBRARIES ${SMTK_LINK_LIBRARIES} -lssh2 -lz -lpthread)
 elseif(CMAKE_SYSTEM_NAME STREQUAL "Windows")
-        set(SMTK_LINK_LIBRARIES ${SMTK_LINK_LIBRARIES} -lwsock32 -lws2_32 -lole32 -limm32 -lwinmm)
-	remove_definitions(-DUNICODE)
+				remove_definitions(-DUNICODE)
```

Comment:
i think the indentation here should be one tab?

also line 93 (`set(SMTK_LINK_LIBRARIES...`) can be changed to use a single tab.